doc: update the ntp configuration (#472)
harvester/docs: docs/host/host.md

KSM starts when the available memory falls below the **Threshold Coefficient**. If it detects that it is running, `pages_to_scan` increments by 300 (**Boost**) every minute until it reaches 5000 (**Maximum Pages**).

KSM will stop when the available memory is above the **Threshold Coefficient**. If it detects that it is stopped, `pages_to_scan` decrements by 100 (**Decay**) every minute until it reaches 1000 (**Minimum Pages**).

## NTP Configuration

> Time synchronization is important for the distributed cluster architecture, so there is a simple way to configure the NTP servers on Harvester.

Reviewer: ```suggestion
Time synchronization is an important aspect of distributed cluster architecture. Because of this, Harvester now provides a simpler way for configuring NTP settings.
```

Reviewer: Because the Harvester UI uses the term "Settings", we should make it easy for users to map the concepts to that particular screen. Here is the wording convention that I used:
- "configure" and "configuration": refer to the action the user performs
- "setting": refers to what the user configures

> In the previous version, we only support NTP configuration when [installation](https://docs.harvesterhci.io/v1.2/install/harvester-configuration#osntp_servers). If you want to change the NTP configuration, manually update the configuration file on each node.

Reviewer 2: ```suggestion
In the previous version, we only support NTP configuration [during the installation process](https://docs.harvesterhci.io/v1.2/install/harvester-configuration#osntp_servers). If you want to change the NTP configuration, it was necessary to manually update the configuration file on each node.
```

> In the previous version, Harvester only supported NTP configuration [during the installation process](https://docs.harvesterhci.io/v1.2/install/harvester-configuration#osntp_servers). If you want to change the NTP configuration, it was necessary to manually update the configuration file on each node.

Reviewer: ```suggestion
In previous Harvester versions, NTP settings were mainly configurable [during the installation process](https://docs.harvesterhci.io/v1.2/install/harvester-configuration#osntp_servers). To modify the settings, you needed to manually update the configuration file on each node.
```

> In v1.2.0, Harvester supports NTP configuration on Harvester/Settings. You can configure NTP servers anytime for the whole Harvester cluster. The NTP configuration will be applied to all nodes in the cluster.

Reviewer 2: ```suggestion
In v1.2.0, Harvester supports NTP configuration in Harvester/Settings. You can configure NTP servers any time for the whole Harvester cluster. The NTP configuration will be applied to all nodes in the cluster.
```

> In v1.2.0, Harvester supports NTP configuration in Harvester/Settings. You can configure NTP servers any time for the whole Harvester cluster. The NTP configuration will be applied to all nodes in the cluster.

Reviewer: ```suggestion
Beginning with version v1.2.0, Harvester is supporting NTP configuration on the Harvester UI Settings screen (**Advanced** > **Settings**). You can configure NTP settings for the entire Harvester cluster at any time, and the settings are applied to all nodes in the cluster.
```

Reviewer: The phrase "configure NTP servers" implies that the user is configuring the actual server, which is not the case here. "NTP settings" covers server addresses and other settings related to the specified servers.

![](/img/v1.3/host/harvester-ntp-settings.png)

> You could setup multiple NTP Servers like the following figure.

Reviewer 2: ```suggestion
You could setup multiple NTP Servers like the following figure:
```

> You could setup multiple NTP Servers like the following figure:

Reviewer: ```suggestion
You can set up multiple NTP servers at once.
```

![](/img/v1.3/host/harvester-ntp-settings-multiple.png)

> You can check the configuration result with CRD `node.harvesterhci.io/ntp-service`

Reviewer 3: ```suggestion
You can check the configuration result from the `node.harvesterhci.io/ntp-service` annotation in Kubernetes Nodes: 
```

> You can check the configuration result from the `node.harvesterhci.io/ntp-service` annotation in Kubernetes Nodes: 
> - `ntpSyncStatus`: synced or unsynced
> - `currentNtpServers`: current NTP servers

Reviewer: ```suggestion
You can check the settings in the `node.harvesterhci.io/ntp-service` annotation in Kubernetes nodes: 
- `ntpSyncStatus`: Status of the connection to NTP servers (possible values: `synced` and `unsynced`)
- `currentNtpServers`: List of existing NTP servers
```

Reviewer: Please check if my understanding is correct. I could not find information on the annotation mentioned here.

```
$ kubectl get nodes harvester-node-0 -o yaml |yq -e '.metadata.annotations.["node.harvesterhci.io/ntp-service"]'
{"ntpSyncStatus":"synced","currentNtpServers":"0.suse.pool.ntp.org 1.suse.pool.ntp.org"}
```

There are points you need to know: 

> 1. Do not directly modify the NTP config on the host. Node-manager will keep the consistency between the configuration on Harvester/Settings and the configuration on the host.

Reviewer 3: ```suggestion
1. Do not modify the NTP config on hosts. Harvester will keep syncing the configuration from Harvester/Settings to hosts.
```

> 2. Suppose your system is upgraded from the previous version. In that case, you need to manually configure your NTP servers on Harvester/Settings first because Harvester does not know your last NTP configuration if there are conflicts. So, for the upgrade case, your ntp-server will be empty, like the following figure.

Reviewer 3: ```suggestion
2. Suppose your system is upgraded from a previous version. In that case, you need to manually configure your NTP servers on Harvester/Settings first because Harvester does not know your last NTP configuration if there are conflicts. So, for the upgrade case, your ntp-server will be empty, like the following figure.
```

> There are points you need to know: 
> 1. Do not modify the NTP config on hosts. Harvester will keep syncing the configuration from Harvester/Settings to hosts.
> 2. Suppose your system is upgraded from a previous version. In that case, you need to manually configure your NTP servers on Harvester/Settings first because Harvester does not know your last NTP configuration if there are conflicts. So, for the upgrade case, your ntp-server will be empty, like the following figure.

Reviewer: ```suggestion
> **Note:**
>
> 1. Do not modify the configuration file on each node. Harvester will automatically sync the settings that you configured on the Harvester UI to the nodes.
> 1. If you upgraded Harvester from an earlier version, the **ntp-servers** list on the Settings screen will be empty (see screenshot). You must manually configure the NTP settings because Harvester is unaware of the previous settings and is unable to detect conflicts.
```

Reviewer: I converted this block of text to a note so that it stands out. I followed the markup described here: https://www.markdownguide.org/hacks/#admonitions
